### ISSUE TITLE
Broadcast completed/failed tasks

### ISSUE THREAD

**Issue by bohdan-shulha** [ptah-sh/ptah-server#10]
Once the task is complete, it is often required to update some properties in the database (e.g. set the docker_id of the entity) or send notifications somewhere.

Need to create a pair of events for each separate task type: ***CompletedEvent and ***FailedEvent.

As we already have the CreateNetworkTask(result) and InitSwarmTask(result), we'll get four events:

- CreateNetworkCompletedEvent
- CreateNetworkFailedEvent
- InitSwarmCompletedEvent
- InitSwarmFailedEvent
